### PR TITLE
Update project mentor list

### LINKED ISSUE #169: Mentor list needs updating
The [list](https://github.com/confidential-computing/governance/blob/main/project-mentors.md#current-mentors) is quite stale (e.g., Veracruz) and has missing entries (e.g., Veraison)

### PR DESCRIPTION
**Pull request by lkatalin**
Fixes #169

Please double check that I have included the correct Github handles for everyone.

Also, it seems to me that SIG/WG should be listed separately, but am open to dissent on this, in which case I can add those.

### CODE REVIEW

**Review by thomas-fossati — APPROVED** [review]
LGTM, thanks!

Question: should the two SIGs (Attestation and Governance) be added too?

**Comment by lkatalin** [comment]
> LGTM, thanks!
> 
> Question: should the two SIGs (Attestation and Governance) be added too?

It's a good question and one I hoped to discuss. Currently the project mentors doc refers to things like the project progression policy, which don't seem to apply to SIGs. We could add them anyway with some more explanation, or put them on another page.

**Review by dthaler — APPROVED** [review]

**Comment by dthaler** [comment]
> > LGTM, thanks!
> > Question: should the two SIGs (Attestation and Governance) be added too?
> 
> It's a good question and one I hoped to discuss. Currently the project mentors doc refers to things like the project progression policy, which don't seem to apply to SIGs. We could add them anyway with some more explanation, or put them on another page.

I agree that the project progression policy doesn't seem to apply to SIGs and so not having a mentor seems ok.

File: project-mentors.md [hunk 1]
```diff
@@ -33,11 +33,10 @@ Responsibilities of the mentor include:
 
 Currently accepted CCC projects have been assigned the following mentors:
 
-* **[CCC-Attestation](https://github.com/CCC-Attestation)**: [Dan Middleton](https://github.com/dcmiddle) and [Aeva Black](https://github.com/AevaOnline)
-* **[Enarx](https://github.com/enarx)**: [Mike Bursell](https://github.com/MikeCamel)
+* **[Enarx](https://github.com/enarx)**: [Nick Vidal](https://github.com/nickvidal)
 * **[Gramine](https://gramineproject.io/)**:[Eric Voit](https://github.com/ericvoit)
```
Comment (thomas-fossati) [inline project-mentors.md:37]: It looks like there's a missing space:

```suggestion
* **[Gramine](https://gramineproject.io/)**: [Eric Voit](https://github.com/ericvoit)
```

**Comment by thomas-fossati** [comment]
> > > LGTM, thanks!
> > > Question: should the two SIGs (Attestation and Governance) be added too?
> > 
> > 
> > It's a good question and one I hoped to discuss. Currently the project mentors doc refers to things like the project progression policy, which don't seem to apply to SIGs. We could add them anyway with some more explanation, or put them on another page.
> 
> I agree that the project progression policy doesn't seem to apply to SIGs and so not having a mentor seems ok.

Works for me.

File: project-mentors.md (hunk 1 above)
Comment (lkatalin) [reply project-mentors.md]: Thanks for the catch. Fixed!

**Review by yashkmankad — APPROVED** [review]

**Pull request merged**